verticapy/pipeline/_helper.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python3
```

Comment:
can you organise this file using comments and creating some sections to explain the kind of functions:

example: 

# Parsing Functions

A-Z order...

# Setup

A-Z order...